css/style.css

```diff
@@ -0,0 +1,173 @@
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+body{
+    font-family: 'Helvetica', 'Arial', sans-serif;
+}
+/*impact*/
+
+/*.container{
+  width:970px;
+  margin:auto;
+}*/
+
+.container{
+  min-width:400px;
+  max-width:1200px;
+  margin: auto;
+}
+/*Can set this way and only have on container for the page*/
+
+header{
+  background-color:black;
+  width:100%;
+  height:100px;
```

Comment:
try not to define heights of container divs
it should naturally be stretched to it's contents :)